packages/alfa-dom/src/node.ts

```diff
@@ -319,33 +325,36 @@ export namespace Node {
     Traversal.nested
   );
 
-  export function from(json: Element.JSON): Element;
+  export function from(json: Element.JSON, device: Option<Device>): Element;
```

Comment:
```suggestion
  export function from(json: Element.JSON, device?: Device): Element;
```
We should make these optional argument (and build the `Option` if needed), in order to not completely break the API.